lib/galaxy/selenium/navigation.yml

```diff
@@ -561,7 +561,9 @@ libraries:
       select_all: '#select-all-checkboxes'
       select_one: '.lib-folder-checkbox'
       empty_folder_message: '.empty-folder-message'
-      btn_open_upper_folder: '.btn_open_folder'
+      btn_open_parent_folder:
+        type: xpath
+        selector: '//a[contains(text(), "${folder_name}")]'
```

Comment:
I've been using data- attributes to avoid this xpath pattern (i.e. https://github.com/galaxyproject/galaxy/pull/11160/files#diff-6383837499b568bc9eafea794ae063d6e8b74061c4bdf5501a4f7e25719f3b14R15). If we're going to use navigation.yml in tours I'm not sure if xpath will be usable (maybe that is not a problem though). I also just like annotated, structured DOM elements - but maybe that is unnecessary markup that could slow things down and shouldn't be a best practice.

Reply:
I see, ok then, I will keep it in mind and thanks for the merge!